### ISSUE TITLE
LayoutLM prediction with score?

### ISSUE THREAD

**Issue by nk-alex** [NielsRogge/Transformers-Tutorials#402]
Taking as starting point this [notebook](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv2/FUNSD/True_inference_with_LayoutLMv2ForTokenClassification_%2B_Gradio_demo.ipynb) more specifically, this function:

```
from PIL import ImageDraw

draw = ImageDraw.Draw(image)

font = ImageFont.load_default()

def iob_to_label(label):
    label = label[2:]
    if not label:
      return 'other'
    return label

label2color = {'question':'blue', 'answer':'green', 'header':'orange', 'other':'violet'}

for prediction, box in zip(true_predictions, true_boxes):
    predicted_label = iob_to_label(prediction).lower()
    draw.rectangle(box, outline=label2color[predicted_label])
    draw.text((box[0]+10, box[1]-10), text=predicted_label, fill=label2color[predicted_label], font=font)

image
```

I would also like to print the prediction score in % format. I know about  `probabilities = torch.nn.functional.softmax(logits, dim=-1)` but I don't quite get how to apply it to obtain the prediction score in % for each true_predictions.

**Comment by nk-alex** [comment]
I think I get the result. Correct me if I'm wrong. From that output, I can extract that there are 14 type of labels and the most likely one is number 13 with 0.992790937423706 (99.2%).

But that is not quite what I was aming for. I'm not looking for the probability of each label for that prediction. I'm looking for the prediction score itself. Something like this prediction has a confidence of 75%

**Comment by tianchiguaixia** [comment]
![image](https://github.com/user-attachments/assets/780569e8-db3e-44bf-aa3a-a593583ca75e)
